services/sasquatch/templates/vault-secret.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: ricoberger.de/v1alpha1
+kind: VaultSecret
+metadata:
+  name: sasquatch
+  namespace: sasquatch
+spec:
+  path: {{ .Values.vaultSecretsPath }}/sasquatch
+  type: Opaque
+---
+apiVersion: ricoberger.de/v1alpha1
+kind: VaultSecret
+metadata:
+  name: pull-secret
+spec:
+  path: {{ .Values.vaultSecretsPath }}/pull-secret
+  type: kubernetes.io/dockerconfigjson
```

Comment:
Ugh, I'm not sure what to think about this. I would rather all the references go through the pull-secret chart for consistency, but I can see how the configuration is a bit more annoying. I think the right solution is to use global variables to affect the pull-secret chart, though, so I think I would still lean towards using the pull-secret chart instead of replicating its contents here.

Reply:
Oh, I thought you would like the idea of eliminating the pull-secret chart entirely :)

Reply:
Hm, actually, maybe that's not a bad idea. Maybe let's talk about this in the next stand-up or co-work.